src/components/DeleteTargetModal/styles.js

```diff
@@ -0,0 +1,56 @@
+import { Dimensions, StyleSheet } from 'react-native';
+
+import { WHITE, YELLOW_TRANSPARENT } from 'constants/colors';
+
+const styles = StyleSheet.create({
+  circleView: {
+    alignSelf: 'center',
+    backgroundColor: YELLOW_TRANSPARENT,
+    borderRadius: 100 / 2,
```

Comment:
🤔 
```suggestion
    borderRadius: 50
```
if that 100 depends on another variable, you should declare it as a constant